vue3_ts/src/core/utils/yup/index.ts

```diff
@@ -0,0 +1 @@
+export * from '@/views/veevalidate-yup/veevalidate-yup.schemas'
```

Comment:
please create a folder called schemas on the same level and principles like for example types